services/workflows-service/src/business/business.controller.external.ts

```diff
@@ -167,19 +167,10 @@ export class BusinessControllerExternal {
       },
     });
 
-    await this.prismaService.$transaction(async transaction => {
-      const stringifiedMetadata = JSON.stringify(updatedMetadata);
-
-      await transaction.$executeRaw`
-        UPDATE "Business"
-        SET "metadata" = jsonb_deep_merge_with_options(
-          COALESCE("metadata", '{}'::jsonb),
-          ${stringifiedMetadata}::jsonb,
-          ${ARRAY_MERGE_OPTION.BY_INDEX}
-                         )
-        WHERE "id" = ${businessId}
-          AND "projectId" = ${currentProjectId};
-      `;
+    await this.businessService.updateById(businessId, {
```

Comment:
Updating has to go through the `BusinessService` to sync the new data with the Unified API.